lib/meta.nix

```diff
@@ -162,5 +162,10 @@ rec {
        getExe' pkgs.imagemagick "convert"
        => "/nix/store/5rs48jamq7k6sal98ymj9l4k2bnwq515-imagemagick-7.1.1-15/bin/convert"
   */
-  getExe' = x: y: "${lib.getBin x}/bin/${y}";
+  getExe' = x: y:
+    if !lib.isDerivation x
+    then throw "A non-derivation value has been passed to lib.getExe"
+    else if builtins.length (lib.splitString "/" y) > 1
+    then throw "A nested path has been passed to lib.getExe"
```

Comment:
```suggestion
    then throw "${y} can not be a path"
```

Reply:
worded this one slightly differently to be more clear but should be good